src/user_default.erl

```diff
@@ -0,0 +1,14 @@
+-module(user_default).
```

Comment:
Is it safe to include a `user_default` in the repo? I mean, we can think about what happens if people are using their own user_defaults and don't want Gradualizer's one to override it. If the one loaded first takes priority, it's safe, right?

Reply:
I'm not quite sure what the best solution is here. I just included it under `src/` because it was convenient but I agree it's not ideal. I suppose we can put it under `examples/` if you think that's better.

Reply:
I don't know. Maybe it's no problem, but I haven't looked into it enough. Maybe src/ is fine but we need to change -pa to -pz in `make shell` or something like that. If there are multiple modules with the same name in the code path, the order in the code path matters...

Reply:
Ok, I've changed it to '-pz' which I think makes sense.